data/unpack.go

```
@@ -21,24 +31,27 @@ func Unpack(base string, uri string) (err error) {
 	}
 
 	if info.IsDir() {
-		os.Mkdir(base, 0777)
-		children, err := file.Readdir(0)
+		err := os.MkdirAll(extractPath, 0777)
```

Switched to using `os.MkdirAll`, to suppress error if path already exists.